src/components/Permissions/DataList.spec.jsx

```diff
@@ -1,7 +1,27 @@
 import { render } from '@testing-library/react'
 import React from 'react'
 import DataList from './DataList'
-import AppLike from 'test/AppLike'
+import { Q, useQuery, isQueryLoading, hasQueryBeenLoaded } from 'cozy-client'
+
+jest.mock('cozy-ui/transpiled/react/I18n/withLocales', () => {
+  return () => Component => {
+    const t = text => text
+    // eslint-disable-next-line react/display-name
+    return () => <Component t={t} />
+  }
```

Comment:
this could be done in one line:

`jest.mock('cozy-ui/transpiled/react/I18n/withLocales', () => () => Component => () => <Component t={text => text}/>)`